### DISCUSSION TITLE
Website domain

### DISCUSSION THREAD

**Discussion by alfonsogarciacaro** [fable-compiler/Fable#3220] [General]
As some of you may remember, we actually don't own the fable.io domain. This is because when we first published the website Tomas Petricek knew the owner of the domain and ask him if we could use it, so we got the permission but not the ownership.

Unfortunately, this is a risk because we can lose the domain at any time. It's also a limitation, for example, I needed a subdomain to set up a Discourse server but I couldn't enable it because I don't have management rights for the domain.

I recently asked the owner if he could transfer the ownership. At first he responded quickly and positively but after he hasn't responded to my follow-up emails so the transfer wasn't complete.

The question then is whether we keep the current situation which actually has been working well for many years (and set up a new domain for the Discourse server) or should we ~~take the Fable 4 release as an opportunity to~~ announce a new domain. Updating all links is going to be a hassle, but I assume we can add redirects to fable.io for the time being.

> Edit: after giving it a thought, if we decide to move to a new domain probably this cannot be complete before Fable 4 release.

cc @MangelMaxime

**Comment by MangelMaxime** [comment]
It would be for the best if we can get the `https://fable.io` domain because changing a domain is difficult.

Especially, if we don't own the old one it means that we can't easily put redirection or that they could be removed in the future if the actual owner change the destination of the domain.

**Comment by MangelMaxime** [comment]
Do you think that this is a lost cause to try get the domain?

Do you think Tomas Petricek could help us on this side, if he still have contact with that person?

**Comment by dbrattli** [comment]
I think we should buy another domain. The `.io` domains are quite expensive i.e `$60` per year.
